content/en/getting_started/serverless/_index.md

```diff
@@ -0,0 +1,145 @@
+  # Entry Lambda Function Code
+  def handler(event, context):
+
+    raise Exception('Throw an error.')
+```
+
+{{< img src="getting_started/serverless/aws_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+
+Deploy this change and invoke your sample app again to see how you can investigate this error in Datadog.
+
+{{< img src="getting_started/serverless/dd_serverless_view_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+Notice that `datadog-sample-entry-function` has five errors.
+
+## Function details
+Click on your function to see more details regarding invocations and recent deployments.
+
+{{< img src="getting_started/serverless/details_error.png" alt="Close-up of two functions" style="width:80%;">}}
+
+The detailed view, as shown above, contains three graphs. You can set these to display any available metric; by default, they show three [enhanced Lambda metrics][7]: invocations, errors, and duration. 
+
+Datadog generates enhanced Lambda metrics out-of-the-box with low latency, several second granularity, and detailed metadata for cold starts and custom tags. You can also view the default [enhanced Lambda metrics dashboard][8].
+
+
+### Invocations
+The **Invocations** tab displays your function’s latest invocations. 
+
+Each invocation is associated with a trace. Click on **Open Trace** to see the trace for each invocation:
+
+{{< img src="getting_started/serverless/dd_flame_graph.png" alt="Close-up of two functions" style="width:80%;">}}
+
+The **Flame Graph** tab shows exactly what happened during the duration of this invocation, including which services had the highest percentage of the total execution time. The flame graph displays the request as it travels from APIGateway, through your `datadog-sample-entry-function`, through SNS, SQS, and finally your `datadog-sample-sqs-function`.
+
+{{< img src="getting_started/serverless/trace_map.png" alt="Close-up of two functions" style="width:80%;">}}
+
+The **Trace Map** tab visualizes the flow of your services and how they connect to each other.
+
+The lower half of the detailed trace view displays a stack trace, which reveals the line of code responsible for throwing the error:
+
+```
+Traceback (most recent call last):
+  File /opt/python/lib/python3.9/site-packages/datadog_lambda/wrapper.py, line 142, in __call__
+    self.response = self.func(event, context, **kwargs)
+File /var/task/index.py, line 17, in handler
+    raise Exception('Throw an error.')
+Exception: Throw an error.
+```
+
+Underneath, you can also examine your Lambda request and response payloads. Datadog collects event payloads for every Lambda invocation.
+
+### Logs
+
+The serverless sample app has logs enabled by default. You can see each function's logs under its **Logs** tab. 
+
+{{< img src="getting_started/serverless/dd_logs_view.png" alt="Close-up of two functions" style="width:80%;">}}
+
+You can filter these logs to only see errors, or view them in the [Log Explorer][6].
+
```

Comment:
As we discussed previously, we should end this guide with showing prospects that they can easily set up a monitor and get a nice-looking alert in slack.